src/oc_erchef/apps/oc_chef_authz/src/oc_chef_authz_db.erl

```diff
@@ -550,17 +531,34 @@ fetch_group_authz_id_sql(#oc_chef_authz_context{server_api_version = ApiVersion,
 %%  * in oc_chef_authz_groups
 %%  * in oc_chef_organization_policy
 %%
-fetch_global_admins(#oc_chef_authz_context{otto_connection=_Server,
-                                           darklaunch = Darklaunch} = Ctx,
-                    OrgName) ->
-    GlobalGroupName = make_global_admin_group_name(OrgName),
+fetch_read_access_group(#oc_chef_authz_context{} = Ctx, OrgName) ->
+    ReadAccessGroupName = make_read_access_group_name(OrgName),
+    case fetch_global_group(Ctx, ReadAccessGroupName) of
+        {not_found, authz_group} ->
```

Comment:
Maybe there should be a note in here that should be removed at some point when compatibility no longer matters?